packages/slinkity/eleventyConfig/applyViteHtmlTransform.js

```diff
@@ -58,7 +58,7 @@ async function handleSSRComments({ content, outputPath, componentAttrStore, vite
       const clientRenderer = rendererMap[rendererName].client
       const loaderScript = toLoaderScript({ componentPath, props, hydrate, id, clientRenderer })
       const attrs = toHtmlAttrString({ [SLINKITY_ATTRS.id]: id })
-      return `<${SLINKITY_REACT_MOUNT_POINT} ${attrs}>\n\t${serverRenderedComponents[id]}\n</${SLINKITY_REACT_MOUNT_POINT}>\n${loaderScript}`
```

Comment:
Removed newlines after hitting a nasty hydration error from Vue: expected `div` but found `" "`. Seems this was from our newlines and tabbing applied here. Removed to un-block hydration!